pkg/storage/bloom/v1/util.go

```diff
@@ -35,7 +35,7 @@ var (
 	// 4KB -> 64MB
 	BlockPool = BytePool{
 		pool: pool.New(
-			4<<10, 64<<20, 4,
+			4<<10, 64<<24, 4,
```

Comment:
IMO `1<<30` would be easier to read
also, the comment above does not match any more